src/test/java/berlin/yuna/nano/services/metric/logic/MetricServiceTest.java

```diff
@@ -12,27 +12,19 @@
 class MetricServiceTest {
 
     @Test
-    void testAddMetricsEndpoint() {
-        final MetricService metricService = new MetricService();
-        Event event = new Event(EVENT_HTTP_REQUEST, Context.createRootContext(), new HttpObject().methodType(HttpMethod.GET).path("/metrics/influx"), null);
-        metricService.addMetricsEndpoint(event);
-        assertThat(event.responseOpt(HttpObject.class)).isPresent();
-        assertThat(((HttpObject) event.payload()).path()).isEqualTo("/metrics/influx");
-        assertThat(((HttpObject) event.payload()).path()).isNotEqualTo("/metrics/prometheus");
-        assertThat(event.response(HttpObject.class).statusCode()).isEqualTo(200);
-    }
-
-    @Test
-    void testAnyMetricsEndpoint() {
-
-        Event event= new Event(EVENT_HTTP_REQUEST, Context.createRootContext(), new HttpObject().methodType(HttpMethod.GET).path("/metrics/influx"), null);
+    void testMetricsEndpoint() {
+        String influxPath = "/metrics/influx";
```

Comment:
We need Integration Tests here as we want to prove the connection between `HttpService` and `MetricService`.
UnitTests should be only used for testing dummy model structure.

### Test with different configurations:
* without basePath
* with basePath
* with prometheus path
* with base path and prometheus path

Integration Test could look like:
```
final Nano nano = new Nano(Map.of(CONFIG_LOG_LEVEL, TEST_LOG_LEVEL), new MetricService(), new HttpService());

final HttpObject result = new HttpObject()
  .methodType(HttpMethod.GET)
  .path("/metrics/prometheus")
  .send(nano.newContext(MetricServiceTest.class));

// TODO: assertions
assertThat(result).isNotNull();
assertThat(result.bodyAsString()).containsString("AA", "BB",...)

nano.shutdown(this.getClass());
assertThat(service.stopCount()).isEqualTo(1);
```